<launch>
  <arg name="robot" default="cyton"/>

  <arg name="group"        default="arm_move_group"/>
  <arg name="end_effector" default="gripper"/>

  <node pkg="moveit_simple_grasps" type="moveit_simple_grasps_server" name="moveit_simple_grasps_server">
    <param name="group"        value="$(arg group)"/>
    <param name="end_effector" value="$(arg end_effector)"/>

    <rosparam command="load" file="$(find cyton_alpha_pick_place)/config/$(arg robot)_grasp_data.yaml"/>
  </node>
</launch>
